django_project/trail/serializers/category.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+from ..models.category import Category
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '01/10/2018'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+class CategorySerializer(serializers.ModelSerializer):
+    """Serializer class for The Category model."""
+
+    class Meta:
+        model = Category
+        fields = ('order', 'image')
```

Comment:
no guid? no name? Order is not a property of the category model I dont think....